persistence/rocksdb/src/main/java/org/infinispan/persistence/rocksdb/internal/RocksDBBlockHoundIntegration.java

```diff
@@ -0,0 +1,20 @@
+package org.infinispan.persistence.rocksdb.internal;
```

Comment:
`impl` package?

Reply:
I personally find it being in a package like `internal` a bit better as it really isn't an implementation class of the public interfaces. I have just repurposed `impl` in some modules since it is private. WDYT?

Reply:
That's a good point. I hadn't realised that we already used `.internal.` packages in some places, so let's stick with that.